Project-I-Social-Card/social-card/src/components/CardComponents/Card.css

```diff
@@ -8,12 +8,16 @@
 .card {
     border:1px solid lightgray;
     width: 80%;
+    border-top-left-radius: .5rem;
+    border-top-right-radius: .5rem;
     border-bottom-left-radius: .5rem;
     border-bottom-right-radius: .5rem;
 }
 
 .card img {
     width:100%;
+    border-top-left-radius: .5rem;
+    border-top-right-radius: .5rem;
 }
 
```

Comment:
- Haja great job on the assingment.
- Your code is clean and You are properly splitting up the code into multiple components. 
- There are only minor issues with styling but that's just simple setting the size of the container to fit the provided design file
- You even completed the stretch task, congrats on that ! 👍 
Your git workflow is fluid and You did amazing job on this assingment 👍